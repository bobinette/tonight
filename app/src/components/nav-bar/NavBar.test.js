import Vue from 'vue';
import Vuex from 'vuex';

import NavBar from './NavBar';

Vue.use(Vuex);

test('render navbar', () => {
  const Constructor = Vue.extend(NavBar);
  const store = new Vuex.Store({
    getters: {
      userid: () => 1,
      username: () => 'test',
    },
  });
  const vm = new Constructor({ store }).$mount();

  expect(vm.$el).toMatchSnapshot();
});
