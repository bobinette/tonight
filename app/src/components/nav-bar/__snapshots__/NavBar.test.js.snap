// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render navbar 1`] = `
<nav
  class="navbar navbar-row navbar-top navbar-inverse bg-primary white"
>
  <div
    class="container"
  >
    <div
      class="row w-100 flex-align-center flex-space-between"
    >
      <h5>
        <img
          height="24"
          src="[object Object]"
          width="24"
        />
        
        Tonight
      
      </h5>
       
      <span>
        <span>
          test
        </span>
         
        <button
          class="btn btn-link white"
        >
          <i
            class="fa fa-sign-out"
          />
        </button>
      </span>
    </div>
  </div>
</nav>
`;
